drivers/soundwire/generic_bandwidth_allocation.c

```diff
@@ -534,14 +535,26 @@ static int sdw_compute_bus_params(struct sdw_bus *bus)
 		}
 	}
 
+	/* FIXME: set default_col with reasonable mechanism */
+	if (curr_dr_freq == 19200000)
+		mstr_prop->default_col = 8;
+	else
+		mstr_prop->default_col = 4;
```

Comment:
@plbossart Do you have a rule of selecting frame shape? Currently, I use a hack to test.

Reply:
> I think you need to start from the root frequency. 19.2: default_row = 50 12: default_row = 125 12.288: default_row = 64
> 
> Then you need to take the bus frequency, multiply it by to to get actual DDR rate, divide it by the frame rate and again by default row and you get the default_col.

Changed to `default_col = curr_dr_freq / m_rt->stream->params.rate / mstr_prop->default_row;`